tests/test_time_series_graph.py

```diff
@@ -581,6 +581,44 @@ def test_from_adjacency_matrices(self):
                 self.assertLess(source.time_lag, destination.time_lag)
                 self.assertEqual(edge.get_edge_type(), EdgeType.DIRECTED_EDGE)
 
+        # test for the bugfix
+        edge_pairs = [
```

Comment:
This may as well be a separate unit test as it is entirely separated from the previous, and keeps it more of a 'unit' test